samples/interactive-tutorials/src/main/java/product/ImportProductsGcs.java

```diff
@@ -37,26 +38,29 @@
 
 public class ImportProductsGcs {
 
-  private static final String PROJECT_ID = System.getenv("PROJECT_ID");
-  private static final String DEFAULT_CATALOG =
-      String.format(
-          "projects/%s/locations/global/catalogs/default_catalog/" + "branches/0", PROJECT_ID);
-  private static final String GCS_BUCKET = String.format("gs://%s", System.getenv("BUCKET_NAME"));
-  private static final String GCS_ERROR_BUCKET = String.format("%s/errors", GCS_BUCKET);
-  private static final String GCS_PRODUCTS_OBJECT = "products.json";
-  // TO CHECK ERROR HANDLING USE THE JSON WITH INVALID PRODUCT
-  // GCS_PRODUCTS_OBJECT = "products_some_invalid.json"
-
   public static void main(String[] args) throws IOException, InterruptedException {
-    ImportProductsRequest importGcsRequest = getImportProductsGcsRequest(GCS_PRODUCTS_OBJECT);
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = ServiceOptions.getDefaultProjectId();
+    String defaultCatalog =
```

Comment:
Same to rename as branchName, probably do it for all the product samples (but not event samples since events' parent resource is catalog, while product's parent is branch).

Reply:
Changed.